src/Features/RoyalShaving/components/StageSection/StageSection.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren, ReactElement } from 'react';
+
+type StageSectionVariant = 'row' | 'rowReverse';
+
+interface StageSectionProps extends PropsWithChildren {
+  variant: StageSectionVariant;
+}
+
+export const StageSection = ({ variant, children }: StageSectionProps): ReactElement => {
+  const classes = {
```

Comment:
same

Reply:
this was Ainura's branch, I just utilized it in my component.